### ISSUE TITLE
Custom template logic hard to follow

### ISSUE THREAD

**Issue by spikefishjohn** [GNS3/gns3-web-ui#1340]
Its very unclear how to create a new custom template. 

In the fat client.

![image](https://user-images.githubusercontent.com/31658656/175344861-4c4056bd-0f3a-4c88-acfe-faed704db00d.png)

![image](https://user-images.githubusercontent.com/31658656/175344962-2e4e0340-b718-4ab1-9849-55107caf39b7.png)

![image](https://user-images.githubusercontent.com/31658656/175345004-e56bdc62-7b65-46ec-94ce-db2fc0d31c48.png)

Now the fat client will take you through a new template wizzard. You can also hit copy for an existing template or you can copy an existing. 

Seems like the only web to do this on the webui is to select an existing Qemu template, select a deploy from server then change the configuration from the 3rd window? I'm not sure if that is how that is intended to work but I don't see any other option to access that.

![image](https://user-images.githubusercontent.com/31658656/175345381-08d89185-34e1-468d-bf80-5b5ab2861e2c.png)

![image](https://user-images.githubusercontent.com/31658656/175345487-2adc777f-e02c-424c-9757-66c64105595b.png)

![image](https://user-images.githubusercontent.com/31658656/175345645-8217bc4a-b36d-4c59-a23e-2cd1070e1946.png)

![image](https://user-images.githubusercontent.com/31658656/175345691-7894a373-ba58-418f-b99c-a842fe82bbfd.png)

**Comment by grossmj** [comment]
>Seems like the only web to do this on the webui is to select an existing Qemu template, select a deploy from server then change the configuration from the 3rd window? I'm not sure if that is how that is intended to work but I don't see any other option to access that.

There is a `Preferences` page to manually add templates however I agree we have some work to do to make all of this more intuitive.
